templates/forum/new_thread.haml

```diff
@@ -171,6 +183,15 @@
         }
       }
 
+      function fetchResources (skills, section, callback) {
+        var params = {"skills[]": skills, "section": section};
```

Comment:
Why "skills[]" ?

Reply:
readability, that's how $.getJSON will parse a list whether you add [] or not but it's good to know how it's sent